src/client/nativeCommunication.ts

```diff
@@ -264,4 +265,15 @@ function reportNativeElementPositionChanges(): void {
 	}
 }
 
-export { ads, slideshow, videos, reportNativeElementPositionChanges };
+function sendTargetingParams(): void {
+	const targetingParams = getTargetingParams();
+	void analyticsClient.sendTargetingParams(targetingParams);
```

Comment:
Will this crash on devices on an older version of Bridget?

Reply:
I think the thrift client/server is built to deal with this.. When I tested it in a browser without any bridget, any other calls still work. Do you think we should start "if versioning" already?
I'm gonna double check it doesn't crash in the simulator too.

Reply:
Just tested 👍 All works well .. didn't crash or break any subsequent bridget calls. Good to know this for when we introduce new methods in newer bridget versions.